### PR TITLE
Dictionary Definition: Remove quotes before passing it to Wordnik

### PR DESCRIPTION
**Pull request by jagtalon**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [x] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{SpiceRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Wordnik doesn't handle queries that have "" in the request and it chokes and turns blue.

For example, `/v4/word.json/#22round%20robbin#22/definitions?includeRelated=true&includeTags=true&limit=3&api_key=(key
removed)&callback=ddg_spice_dictionary_definition`

This strips out the quotes.

---

Instant Answer Page: https://duck.co/ia/view/dictionary_definition

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by jagtalon** [comment]
I'm the maintainer of this IA. :)

File: lib/DDG/Spice/Dictionary/Definition.pm [hunk 1]
```diff
@@ -3,6 +3,7 @@ package DDG::Spice::Dictionary::Definition;
 
 use strict;
 use DDG::Spice;
+use Text::Trim;
```
Comment (moollaza) [inline lib/DDG/Spice/Dictionary/Definition.pm]: Is this necessary?

**Comment by moollaza** [comment]
LGTM 👍 

@jagtalon Not sure why `Text::Trim` was added but not used though?

File: lib/DDG/Spice/Dictionary/Definition.pm (hunk 1 above)
Comment (MariagraziaAlastra) [inline lib/DDG/Spice/Dictionary/Definition.pm]: Won't a regular regex be enough in this case?

**Comment by jagtalon** [comment]
@moollaza @MariagraziaAlastra thanks for catching that--removed!

File: lib/DDG/Spice/Dictionary/Definition.pm (hunk 1 above)
Comment (jagtalon) [inline lib/DDG/Spice/Dictionary/Definition.pm]: @moollaza @MariagraziaAlastra this isn't necessary--I just forgot to remove it. :)

**Pull request merged**